integration/kubernetes/helm-chart/alluxio/values.yaml

```diff
@@ -197,6 +197,38 @@ jobWorker:
   # JVM options specific to the jobWorker container
   jvmOptions:
 
+
+## Proxy ##
+proxy:
+  enabled: false # Enable this to enable the proxy for REST API
+  env:
+    # Extra environment variables for the worker pod
+    # Example:
+    # JAVA_HOME: /opt/java
+  args:
+    - proxy
+  # Properties for the proxy component
+  properties:
+  resources:
+    limits:
+      cpu: "4"
+      memory: "4G"
+    requests:
+      cpu: "1"
+      memory: "2G"
```

Comment:
Does the proxy require this much resources?

Reply:
Probably not, but I haven't worked with the proxy long enough to be sure yet, so playing it safe.

Reply:
It'll be great if you could try to shrink it to 1G memory or below. Will you be able to do a quick test with 1G to see if that works? 4G/2G is a bit demanding for many deployments.